src/components/DpAutocomplete/DpAutocomplete.vue

```diff
@@ -1,114 +1,396 @@
 <template>
-  <vue-omnibox
-    :box-height="height"
-    :disable-search="true"
-    :label="label"
-    :options="options"
-    :placeholder="placeholder"
-    :value="currentQuery"
-    @input="emitAndFetch"
-    @searched="emit($event, 'searched')"
-    @selected="emit($event, 'selected')">
-    <template
-      v-for="(_, slotName) in $slots"
-      v-slot:[slotName]="slotData">
-      <slot
-        v-bind="slotData"
-        :name="slotName" />
-    </template>
-  </vue-omnibox>
+  <div class="relative">
+    <!-- Search container -->
+    <div
```

Comment:
i am curious, why is a div used here instead of a native input? A input already provides styling and built-in behaviors like text selection, clipboard and so on. For example i can not highlight or copy the text in the search field, which can be problematic...

Reply:
It was like that in the old vue-omnibox. There where several issues, trying it with an input-element, so I decided to stick with the already working approach. Even though it comes with drawbacks. I love to find out whats going on, so we can use the native element, but not now :-/